src/overview/reducer.js

```diff
@@ -5,13 +5,20 @@ import * as actions from './actions'
 const defaultState = {
     searchResult: {rows: []},
     query: '',
+    nlp_date:'',
     waitingForResults: 0,
+    startDate: '', 
```

Comment:
I would expect those to be null or undefined since they are dates after. If they are string in 
```
+    onStartDateChange: date => {
 +             dispatch(actions.handleStartChange({startDate: date}))
 +      },
```
I would expect to be called differently than date